src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -500,13 +517,16 @@ private protected override void SaveCore(ModelSaveContext ctx)
             base.SaveCore(ctx);
 
             ctx.SaveStringOrNull(_trainLabelColumn);
+            ctx.SaveStringOrNull(_scoreColumn);
+            ctx.SaveStringOrNull(_predictedLabelColumn);
         }
 
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "MC  PRED",
-                verWrittenCur: 0x00010001, // Initial
+                //verWrittenCur: 0x00010002, // Initial
```

Comment:
(nit)

```
//verWrittenCur: 0x00010001, // Initial
```

Reply:
yep, I just addressed that in my latest commit.

---
In reply to: [339233591](https://github.com/dotnet/machinelearning/pull/4372#discussion_r339233591) [](ancestors = 339233591)